README.md

```diff
@@ -297,6 +299,68 @@ and then change its colors, the items missed would fallback to use the default c
 Please also notice that an empty theme is **NOT** supported due to
 [a bug in serde lib](https://github.com/dtolnay/serde-yaml/issues/86).
 
+### Icon Theme
+
+Icon theme can be configured in a fixed location, `$XDG_CONFIG_DIR/lsd/icons.yaml`,
+for example, `~/.config/lsd/icons.yaml` on macOS,
+please check [Config file location](#config-file-location) to make sure where is `$XDG_CONFIG_DIR`.
+
+As the file name indicated, the icon theme file is a `yaml` file.
+
+Check [Icon Theme file content](#icon-theme-file-content) for details.
+
+#### Icon Theme file content
+
+lsd support 3 kinds of icon configuration:
+- filetype
+- name
+- extension
+
+The default icon theme scheme shipped with `lsd` can be check in [icon theme source code](src/theme/icon.rs), we will load the default theme, and overwrite it with user defined parts, here is a example for icon theme.
+
+lsd icon theme support both nerd font and Unicode in the same time, you can use any one to config the theme, or even combine them.
```

Comment:
"combine them" might be a bit confusing, how about we say something like, "this is the default but you can customize this to use glyphs from nerd-font or unicode emojis" and give the example that you have there.